### DISCUSSION TITLE
Why the fork?

### DISCUSSION THREAD

**Discussion by BenKnigge** [BenKnigge/sqly#7] [Announcements]
It appears that sqlx has been abandoned and yet there are updates and bug fixes that are required to be made.

I had started by merging some of the many open pull requests and then decided to revert those changes until I'm able to get the existing tests to run in github actions. 

Once that's done I'll likely go start the process of merging the existing pr. If anyone would be interested in helping me or is able to convince https://github.com/jmoiron to allow them to start maintaining sqlx that would likely be a superior solution. 

For now I've simply renamed the package to sqly "sql-y" as y follows x and updated the DB drivers to the latest versions.

**Comment by kmpm** [comment]
This fork seems to be abandoned as well, how ironic ;-)

What's your guesstimated timeline on when stuff will start to get merged?
